scripts/pylint_extensions.py

```diff
@@ -134,6 +139,33 @@ def visit_call(self, node):
                             func_name))
                     num_positional_args_unused -= 1
 
+            #Executed if name is not having any default value in the function
+            #definition
+            else:
+                if name is None:
```

Comment:
L145-148 is repetition. Please find a way to do away with the repetition.

Reply:
Done.